nodejs/webrtc-hello-video/index.js

```diff
@@ -0,0 +1,168 @@
+const BandwidthWebRTC = require("@bandwidth/webrtc");
+const express = require("express");
+const uuid = require("uuid");
+const dotenv = require("dotenv").config();
+
+const bodyParser = require("body-parser");
+const app = express();
+app.use(bodyParser.json());
+app.use(express.static("public"));
+
+// global vars
+const port = 3000;
+const accountId = process.env.ACCOUNT_ID;
+var rooms_db = new Map();
+
+BandwidthWebRTC.Configuration.basicAuthUserName = process.env.USERNAME;
+BandwidthWebRTC.Configuration.basicAuthPassword = process.env.PASSWORD;
+var webRTCController = BandwidthWebRTC.APIController;
+
+// app.get("/", (req, res) => {
+//   res.send("Hello World!");
+// });
+
+app.post("/startCall", async (req, res) => {
```

Comment:
This is nit-picky but /joinCall is possibly more accurate for what the endpoint is doing